.eslintrc

```diff
@@ -81,96 +95,97 @@
       0,
       "never"
     ],
-    "flowtype/type-id-match": [
-      0,
-      "^([A-Z][a-z0-9]+)+Type$"
-    ],
+    "flowtype/type-id-match": 0,
     "flowtype/union-intersection-spacing": [
-      1,
+      2,
       "always"
     ],
     "flowtype/use-flow-type": 0,
-    "flowtype/valid-syntax": 0,
 
     ////////// Possible Errors //////////
 
-    "comma-dangle": 1,         // disallow trailing commas in object literals
-    "no-cond-assign": 0,          // disallow assignment in conditional expressions
+    "comma-dangle": [             // disallow trailing commas in object literals
+      2,
```

Comment:
Let's add a prettier config to `package.json` and enable `trailingComma` I don't want to have to force folks to do this manually

Reply:
I'll take a look, `eslint --fix` also does it automatically